addons/web/static/src/scss/bootstrap_review_backend.scss

```diff
@@ -75,7 +75,7 @@
         }
 
         &:not(:first-child) {
-            margin-left: $btn-border-width;
+            margin-left: var(--btn-group-gap, 1px);
```

Comment:
I prefer if we still refer to `$btn-border-width` rather then using the `1px`arbitrary value